### ISSUE TITLE
Escape package_name

### ISSUE THREAD

**Issue by mtasaka** [ruby-gnome2/native-package-installer#8]
Since for Fedora (and Redhat) package, if it contains pkgconfig file foo.pc, it also provides "pkgconfig(foo)" virtual provides, so for ruby-gnome2 dependency-check, I want to use such virtual provides instead of direct package name, like:
```
diff --git a/goffice/dependency-check/Rakefile b/goffice/dependency-check/Rakefile
index 3a755eb54..a8639e2fb 100644
--- a/goffice/dependency-check/Rakefile
+++ b/goffice/dependency-check/Rakefile
@@ -48,6 +48,7 @@ namespace :dependency do
     unless PKGConfig.check_version?("libgoffice-0.10")
       goffice010_spec = {
         :debian => "libgoffice-0.10-dev",
+        :redhat => "pkgconfig(libgoffice-0.10)",
         :homebrew => "goffice",
       }
       unless NativePackageInstaller.install(goffice010_spec)
```
, however with native-package-installer 1.0.5, this causes error like:

```
[tasaka1@localhost ruby-gnome2]$ ruby run-test.rb 
--------------------------------------------------------------------------------
2017-12-03T16:12:43+09:00: Running test for /home/tasaka1/rpmbuild/TEMP/ruby-gnome2/git/ruby-gnome2/goffice
--------------------------------------------------------------------------------
installing 'pkgconfig(libgoffice-0.10)' native package... sh: -c: 行 0: 予期しないトークン `(' 周辺に構文エラーがあります
sh: -c: 行 0: `/bin/sudo -p \[sudo\]\ password\ for\ \%u\ to\ install\ \<pkgconfig\(libgoffice-0.10\)\>:\  dnf install -y pkgconfig(libgoffice-0.10)'
installing 'pkgconfig(libgoffice-0.10)' native package... failed
failed
Failed to run '/bin/sudo -p \[sudo\]\ password\ for\ \%u\ to\ install\ \<pkgconfig\(libgoffice-0.10\)\>:\  dnf install -y pkgconfig(libgoffice-0.10)'.
Failed to run '/bin/sudo -p \[sudo\]\ password\ for\ \%u\ to\ install\ \<pkgconfig\(libgoffice-0.10\)\>:\  dnf install -y pkgconfig(libgoffice-0.10)'.
--------------------
```

So I guess ``package_name`` in native-pacakge-installer.rb must be escaped somehow. Currently I have to write:
```
diff --git a/goffice/dependency-check/Rakefile b/goffice/dependency-check/Rakefile
index 3a755eb54..a8639e2fb 100644
--- a/goffice/dependency-check/Rakefile
+++ b/goffice/dependency-check/Rakefile
@@ -48,6 +48,7 @@ namespace :dependency do
     unless PKGConfig.check_version?("libgoffice-0.10")
       goffice010_spec = {
         :debian => "libgoffice-0.10-dev",
+        :redhat => "pkgconfig\\\(libgoffice-0.10\\\)",
         :homebrew => "goffice",
       }
       unless NativePackageInstaller.install(goffice010_spec)
```

**Comment by kou** [comment]
Thanks for your report. I've fixed and released 1.0.6. Please try it.

**Comment by mtasaka** [comment]
Thank you! 1.0.6 is working. Now I will try to modify ruby-gnome2...
